.github/workflows/index.yml

```diff
@@ -0,0 +1,40 @@
+---
+name: Indexing
+on:
+  schedule:
+    - cron: '42 20 * * SAT'
```

Comment:
Remember to change weekly cron to monthly, especially for repos that don't get a lot of changes.

Reply:
It's up-to project owners. I'm just adding a "template" workflow.

Reply:
Changed to monthly.